src/index.edge.ts

```diff
@@ -1,4 +1,103 @@
+import "performance-polyfill";
```

Comment:
This looks great. But now I'm wondering why not just use this implementation for both "edge" and "node"? Why have two if this one works?

Reply:
There's some NodeJS specific hooks that the node implementation uses (listen for events on `process`, instrument `http` and `https` to trace remote traffic through those libs).  It might make sense to keep those incase there are apps that use those libs?

Reply:
Oh, yeah. Then certainly. Though it's not great that we intercept (and trace) `fetch` in Next.js and `http` by Otel. IMHO we should just support `http` interception in Next.js as well. But in the meantime, this is fine to leave as is.